app/models/user.rb

```diff
@@ -1,10 +1,25 @@
 class User < ActiveRecord::Base
-
   has_secure_password
 
+  attr_accessor :current_password
```

Comment:
It was required when the profile edit update action is triggered and led to model update.